commands/request-listener.ts

```diff
@@ -0,0 +1,37 @@
+import { CENNZNET_NETWORK, ETHEREUM_NETWORK } from "@/libs/constants";
+import { collectPendingRequestIds } from "@/libs/utils/collectPendingRequestIds";
+import { getCENNZnetApi } from "@/libs/utils/getCENNZnetApi";
+import { getLogger } from "@/libs/utils/getLogger";
+import { getRabbitMQSet } from "@/libs/utils/getRabbitMQSet";
+import { AMQPError } from "@cloudamqp/amqp-client";
+import chalk from "chalk";
+
+const logger = getLogger("RequestListener");
+logger.info(
+	`Start RequestListener with CENNZnet: ${chalk.magenta(
+		"%s"
+	)} | Ethereum: ${chalk.magenta("%s")}...`,
+	CENNZNET_NETWORK,
+	ETHEREUM_NETWORK
+);
+Promise.all([getCENNZnetApi()])
```

Comment:
why Promise.all for one promise?

Reply:
ah just for aesthetic and future proof in case we need more things in the future